integration-libs/opf/root/facade/opf-checkout.facade.ts

```diff
@@ -7,17 +7,26 @@
 import { Injectable } from '@angular/core';
 import { facadeFactory, QueryState } from '@spartacus/core';
 import { Observable } from 'rxjs';
-import { OPF_CORE_FEATURE } from '../feature-name';
-import { PaymentInitiationConfig, PaymentSessionData } from '../model';
+import { OPF_FEATURE } from '../feature-name';
+import {
+  OpfPaymentVerificationPayload,
+  OpfPaymentVerificationResponse,
+  PaymentInitiationConfig,
+  PaymentSessionData,
+} from '../model';
 import { ActiveConfiguration } from '../model/opf-checkout.model';
 
 @Injectable({
   providedIn: 'root',
   useFactory: () =>
     facadeFactory({
       facade: OpfCheckoutFacade,
-      feature: OPF_CORE_FEATURE,
-      methods: ['getActiveConfigurationsState', 'initiatePayment'],
+      feature: OPF_FEATURE,
```

Comment:
I changed OPF_CORE_FEATURE to OPF_FEATURE and registered OPF_FEATURE in opf-feature.module.
If I register OPF_CORE_FEATURE in opf-feature.module I get 'adapter not found' in runtime.

Reply:
For this I think it should `OPF_FEATURE`. I don't see a purpose of `OPF_CORE_FEATURE` atm.